templates/studygroups/team_form.html

```diff
@@ -1,42 +1,43 @@
 {% extends 'base.html' %}
 {% load crispy_forms_tags %}
-{% load static %}
 {% load i18n %}
 
 {% block content %}
-<div class="container my-5">
-  <div class="row mb-3">
+    <div class="container my-5">
+      <div class="row mb-3">
 
-    <div class="col-12">
-      <h2>
-        {% blocktrans with team_name=object.name %}
-            Edit {{ team_name }}
-        {% endblocktrans %}
-      </h2>
+        <div class="col-12">
+          <h2>
+            {% blocktrans with team_name=object.name %}
+                Edit {{ team_name }}
+            {% endblocktrans %}
+          </h2>
 
 
-      <form action="" method="POST" enctype="multipart/form-data">
-      {% csrf_token %}
-      {{ form|crispy }}
-        <p><input type="submit" class="btn p2pu-btn blue" /></p>
-    </form>
+          <form action="" method="POST" enctype="multipart/form-data">
+          {% csrf_token %}
+          {{ form|crispy }}
+            <p><input type="submit" class="btn p2pu-btn blue" /></p>
+        </form>
 
-    </div>
-  </div>
-
-  <div class="row">
-    <div class="col-12">
-      <h3>
-        {% trans "Default 'About Us' Text" %}
-      </h3>
-        <p>
-          {% blocktrans with team_name=object.name location=object.location %}
-          {{ team_name }} is proud to partner with P2PU to bring learning circles to our community. Join your neighbors and peers as we gather to learn something together. Interested in creating your own learning circle? Check out our facilitation resources at <a href="https://www.p2pu.org/en/facilitate/">www.p2pu.org/en/facilitate</a> to learn more.
-          {% endblocktrans %}
-        </p>
-    </div>
-  </div>
+        </div>
+      </div>
 
-</div>
+    </div>
+    {% if not object.intro_text %}
+      <script type="text/javascript">
```

Comment:
the TinyMCE widget doesn't accept the `initial` attribute, so i'm setting the default content via javascript.

Reply:
Weird? I didn't look into it, but it does get text from an instance, so there is another way of getting it data to display?